pyha_analyzer/train.py

```diff
@@ -282,25 +287,27 @@ def main(in_sweep=True) -> None:
     
     logger.info("Training...")
     early_stopper = EarlyStopper(patience=cfg.patience, min_delta=cfg.min_valid_map_delta)
-    
-    # MAIN LOOP
-    # pylint: disable-next=global-statement
-    global EPOCH
-    # pylint: disable-next=global-statement
-    global BEST_VALID_MAP
-    EPOCH = 0
-    BEST_VALID_MAP = 0
-
-    for _ in range(cfg.epochs):
-        logger.info("Epoch %d", EPOCH)
-
-        train(model_for_run, train_dataloader, val_dataloader, optimizer, scheduler)
-        EPOCH += 1
-        valid_map = valid( model_for_run, val_dataloader, EPOCH)
-        logger.info("Best validation map: %f", BEST_VALID_MAP)
+
+    best_valid_map = 0.0
```

Comment:
reset epoch here as well

Reply:
Epoch is now defined as a variable in the for loop so it doesn't need to be reset